src/sentry/web/client_config.py

```diff
@@ -168,6 +172,7 @@ def get_client_config(request=None):
         _delete_activeorg(session)
 
     context = {
+        "customerDomain": customer_domain,
```

Comment:
Should we be including the `sentryUrl` and `organizationUrl`? Having consistency between the initial page payload and the responses we get from the organization details API could be beneficial.

Reply:
@markstory I've included them. I won't validate the existence of the org (nor the customer domain feature) so as to avoid enumeration attacks.